rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-upload-to-google-drive.adoc

```diff
@@ -0,0 +1,64 @@
+= Upload to Google Drive 
+
+Upload files to your Google Drive. 
+
+You can export Google Docs, Sheets, and Slides into a selection of supported formats including PDF.
+
+== Before You Begin 
+
+Before uploading a file, ensure you: 
+
+* Configured your Google API Service Account Credentials.
++
+You can configure your credentials as an xref:toolbox-variable-handling-activity-parameters.adoc#defining-activity-parameters[Activity Parameter] or inline in the action step wizard. 
+
+== Properties 
+//OAuth common properties
+include::partial$oauth-authentication-settings.adoc[]
+
+* *Common* 
++
+See xref:common-properties.adoc[Common Properties]
+
+* *File Operation Settings*
+** *Directory path* 
++
+Path to the directory where the file to upload is. 
++
+You can specify either an absolute path or a relative to the Base Path. 
+** *File name*
++
+Name of the file to upload. 
+** *Google Drive Folder ID*
++
+ID of the destination Google Drive folder where to upload the file. 
++
+You can obtain the folder ID by navigating to said Google Drive folder and extracting the ID from the URL (the ID is the value `folder/`). If you don't specify a folder ID, this action step upload the file to the root folder. 
```

Comment:
We should add here that users can also obtain the folder ID by using the Search in Google Drive action step and "Search for: Folders" there, they will get Folder IDs matching their search criteria.